BaseScript/fabfile.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import with_statement
+from fabric.api import env
+from fabric.api import local, settings, abort, run, cd
+from fabric.operations import run, put, sudo
+import sys
+
+
+def installGit():
```

Comment:
Use underscore convention for naming. `install_git()`. since most of the in-built packages have this format as well.